src/Administration/Resources/app/administration/src/module/sw-settings-seo/component/sw-seo-url/index.js

```diff
@@ -210,12 +232,12 @@ Component.register('sw-seo-url', {
 
             if (!currentSeoUrl) {
                 const entity = this.seoUrlRepository.create(Shopware.Context.api);
-                entity.foreignKey = this.defaultSeoUrl.foreignKey;
+                entity.foreignKey = this.foreignKey;
```

Comment:
This could lead to breaks. I would prefer:
```
entity.foreignKey = this.foreignKey !== null ? this.foreignKey : this.defaultSeoUrl.foreignKey;
```

Reply:
The same with route and pathInfo below

Reply:
True. I added the fallback to `defaultSeoUrl` for the three properties.